tools/heavy_measurement_attendance.py

```
@@ -0,0 +1,162 @@
+            attended[lease_class] = str(path)
+    return attended, testimony
+
+
+def workflow_runs(commit, repo=None):
+    """Fall back to the run list. `cancelled` here is the eviction signature."""
+    argv = ["gh", "run", "list", "--commit", commit, "--limit", "100",
+            "--json", "name,status,conclusion,databaseId,workflowName"]
+    if repo:
+        argv += ["--repo", repo]
+    try:
+        completed = subprocess.run(argv, capture_output=True, text=True, check=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)